etc/ci/upload_nightly.sh

```diff
@@ -48,7 +48,10 @@ main() {
     elif [[ "${platform}" == "macbrew" ]]; then
         extension=tar.gz
         package=target/release/brew/*."${extension}"
-    elif [[ "${platform}" == "windows" ]]; then
+    elif [[ "${platform}" == "windows-gnu" ]]; then
```

Comment:
You can use `[[ "${platform}" == "windows-gnu" || "${platform}" == "windows-msvc" ]]`, if these will continue to be the same and it fits on one line.